feature/github-repo-importer/pkg/github/constants.go

```diff
@@ -33,6 +33,15 @@ const (
 
 	DefaultPageSize = 100
 
+	// Feature flags
+	// All feature flags follow the pattern: feature_<feature_name>
+	// Add new features here and they'll automatically work with the config system
```

Comment:
This is not correct information. Please remove it. It's also affected by the comment https://github.com/G-Research/github-terraformer/pull/28/files#r2549525571